deploy_polygon/200_defi/310_erc20_reward_pool.ts

```diff
@@ -11,7 +11,7 @@ const func: DeployFunction = async function (
 
   await deployments.deploy('ERC20RewardPool', {
     from: deployer,
-    contract: 'ERC20RewardPool',
+    contract: 'ERC20RewardPoolV2',
```

Comment:
You kept the same deployment name "ERC20RewardPool"
But what about the former contract ERC20RewardPool we already deployed ? The deploy script won't match the deployment json file

Reply:
which seems to imply we need 2 different deploy scripts ? :(